test/TEST-08-ISSUE-2730/test.sh

```diff
@@ -23,8 +23,11 @@ check_result_qemu() {
 }
 
 test_run() {
-    run_qemu || return 1
-    check_result_qemu || return 1
+    if run_qemu; then
+        check_result_qemu || return 1
+    else
```

Comment:
This doesn't work:
`./TEST-08-ISSUE-2730/test.sh` and `./TEST-09-ISSUE-2691/test.sh` use `QEMU_TIMEOUT`
So, `run_qemu` runs something like this:
```
    if [[ "$QEMU_TIMEOUT" != "infinity" ]]; then
        QEMU_BIN="timeout --foreground $QEMU_TIMEOUT $QEMU_BIN"
    fi
```
The result should be `0`.